src/System.Runtime.Analyzers/Core/ProvideCorrectArgumentsToFormattingMethods.cs

```diff
@@ -86,7 +86,7 @@ public override void Initialize(AnalysisContext analysisContext)
 
                     // params case
                     var paramsArgument = invocation.ArgumentsInParameterOrder[info.FormatStringIndex + 1];
-                    if (paramsArgument.Kind != ArgumentKind.ParamArray)
+                    if (paramsArgument.ArgumentKind != ArgumentKind.ParamArray)
```

Comment:
/cc @genlu @nguerrera @JohnHamby - this file has all the changes I had to make to respond to API changes.